qemu/tests/qemu_guest_agent.py

```diff
@@ -2043,6 +2020,95 @@ def result_check(flag, write_timeout, session):
                               "stamp in guest file." % write_timeout)
         result_check("thaw", write_timeout, session)
 
+    @error_context.context_aware
+    def gagent_check_fstrim(self, test, params, env):
+        """
+        Execute "guest-fstrim" command via guest agent.
+
+        Steps:
+        1) boot up guest with scsi backend device
+        2) init and format the data disk
+        3) get the original blocks of data disk
+        4) create fragment in data disk
+        5) check the used blocks
+        6) execute fstrim cmd via guest agent
+        7) check if the used blocks is decreased
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        def get_blocks():
+            """
+            Get the used blocks of data disk.
+            :return: the used blocks
+            """
+            blocks = process.system_output("stat -t %s" % stg_image_filename)
+            return blocks.strip().split()[2]
```

Comment:
@xiagao,  This command can get real data size for block device or file image?

Reply:
file image according to polarion steps.